modules/installing-with-usb-media.adoc

```diff
@@ -0,0 +1,29 @@
+// This is included in the following assemblies:
+//
+// installing_on_prem_assisted/assisted-installer-installation.adoc
+
+:_content-type: PROCEDURE
+[id="installing-with-usb-media_{context}"]
+= Creating a bootable ISO image on a USB drive
+
+You can install software using a bootable USB drive that contains an ISO image. Booting the server with the USB drive prepares the server for the software installation.
```

Comment:
Fixed. This was a reused module. I removed bootable reference, but stuck with "boot drive," because disk implies spinning media and USB drives do not spin.